examples/plugins/filecount.pp

```diff
@@ -7,11 +7,11 @@
   },
 }
 
-collectd::plugin::filecount::directory { 'foodir':
+collectd::plugin::filecount::directory {'foodir':
```

Comment:
I think out linter plugin requires the whitespace here?

Reply:
yes! I thought I had that commit removed. This has nothing to do with the original change and was just a try to make CI pass.